### ISSUE TITLE
Cannon ammo count doesn't appear sometimes

### ISSUE THREAD

**Issue by Treedot** [runelite/runelite#5418]
If you place your cannon down and THEN activate the cannon plugin, it won't show the ammo count on your cannon. You have to pick it up and place it again or turn on the cannon plugin before placing it.

**Comment by deathbeam** [comment]
Yeah, this is not solveable as there is no way to retrospectively determine which cannon is yours
